docs/sources/installation/configuration.md

```diff
@@ -81,6 +81,32 @@ export GF_AUTH_GOOGLE_CLIENT_SECRET=newS3cretKey
 export GF_PLUGIN_GRAFANA_IMAGE_RENDERER_RENDERING_IGNORE_HTTPS_ERRORS=true
 ```
 
+## Variable expansion
+
+> Only available in Grafana 7.1+.
+
+If any of your options contains the expression `$__<provider>{<argument>}`
+or `${<environment variable>}` they will be processed by Grafana's
+variable expander which will run the provider with the provided argument
+to get the final value of the option.
+
+There are two providers, `env` and `file`.
```

Comment:
```suggestion
There are two providers: `env` and `file`.
```

Reply:
I recommend adding an h3 before each of the two following paragraphs so that users can easily skim to find the provider. Maintain parallelism where possible, so each should have an example.

That said, does this information about providers really beling in configuration.md? Should it be somewhere else in the documentation, or possibly in its own topic in the administration section?

Reply:
Good idea!

🤔 It is specifically for `grafana.ini`, I'm not sure where else in the documentation we'd put that information.

Its own topic could be an alternative, I have a personal preference for documentation on a single long page where I can navigate with ctrl+f. That might be showing here. I wouldn't know what to call it though, "Templating the configuration file" seems too long. "Options providers" sounds obscure and uninteresting.